src/solver/mod.rs

```diff
@@ -1002,6 +1067,11 @@ impl<D: DependencyProvider, RT: AsyncRuntime> Solver<D, RT> {
         while let Some(decision) = self.decision_tracker.next_unpropagated() {
             let pkg = decision.solvable_id;
 
+            tracing::trace!(
+                "Exploring watched solvables for {}",
+                pkg.display(self.provider())
+            );
```

Comment:
I ran a little test and this popped-up quite a lot, is this useful to see in the logs?